crates/duty-tracker/src/lib.rs

```diff
@@ -0,0 +1,131 @@
+mod tx_driver;
+use std::{
+    collections::VecDeque,
+    future::Future,
+    pin::pin,
+    sync::{Arc, Mutex, MutexGuard},
+};
+
+use bitcoin::{
+    opcodes::all::OP_RETURN, Amount, Network, Script, ScriptBuf, Transaction, XOnlyPublicKey,
+};
+use btc_notify::client::BtcZmqClient;
+use futures::{poll, FutureExt, Stream, StreamExt};
+use tokio::task::JoinHandle;
+
+const DUST_VALUE: Amount = Amount::from_sat(330);
```

Comment:
Ugh, I am also using that `const` in https://github.com/alpenlabs/strata-bridge/blob/437f2db78b630898c17e8cbbf305ebe7c06e3678/crates/stake-chain/src/transactions/constants.rs?plain=1#L29-L30

Maybe we should move this to a `primitives` crate? What you think?